packages/KTable/KTable.vue

```diff
@@ -53,19 +53,27 @@
  */
 export const defaultSorter = (key, previousKey, sortOrder, items) => {
   let comparator = null
-
   const type = typeof items[0][key]
 
   if (key !== previousKey) {
-    if (type === 'string') {
+    if (type === 'number') {
+      comparator = (a, b) => a[key] && b[key] && a[key] - b[key]
+    } else {
       comparator = (a, b) => {
-        if (a[key] < b[key]) return -1
-        if (a[key] > b[key]) return 1
+        const transformer = (val) => {
+          if (val === undefined || val === null) {
+            return ''
+          }
+
+          if (Array.isArray(val)) {
+            return String(val[0])
```

Comment:
the first item in the array could also be of any type e.g. undefined, null, number, string.